spec/AutomatedLinkAcceptanceProxy+forwardings.yaml

```diff
@@ -398,8 +398,8 @@ forwardings:
       - server-name: /v1/provide-acceptance-data-of-link-endpoint
         uuid: alap-1-0-1-op-s-is-000
     consequent-requests:
-      - client-name: MWDI://core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/layer-protocol={local-id}/air-interface-2-0:air-interface-pac/air-interface-configuration?fields=air-interface-name
-        uuid: alap-1-0-1-op-c-is-mwdi-1-0-0-211
+      - client-name: MWDI://core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/ltp-augment-1-0:ltp-augment-pac?fields=external-label
+        uuid: alap-1-0-1-op-c-is-mwdi-1-0-0-201
 
   - forwarding-name: RequestForProvidingAcceptanceDataCausesReadingCapabilitiesFromCache
     uuid: alap-1-0-1-op-fc-is-102
```

Comment:
line 461: RequestForProvidingAcceptanceDataCausesAnalysingTheAggregation :: the consequent-request , field shall be updated to original-ltp-name;external-label

Reply:
Fields filter has been updated in forwarding